main.py

```diff
@@ -292,7 +292,7 @@ def update_user_report():
     if not data or not data.get("report_id"):
         return {"error": "Missing report_id"}, 400
 
-    if data.get('status'):  # Only admins can update the status
+    if data.get('self_report_status'):  # Only admins can update the self_report_status
```

Comment:
this is create report. how can admin change the status using create?
should do this change in updateUserreport.